### PR TITLE
- Issue #4 - Add SplashScreen with progress

### PR DESCRIPTION
**Pull request by galihsatriawan**
## Description
- Description of issue
 Implement Splash Screen
- Output before solved
- Changes in Issue
Have Implemented Splash Screen
- Output of solved issue

chat

## Type of Changes?
Just put an x in the [ ] which are valid.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing Suites

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes.
- [ ] ./gradlew assembleDebug assembleRelease

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by iadeelzafar** [comment]
@galihsatriawan Thanks for working on the ticket. Can you please add a screenshot to the splash screen after the changes on the PR description?

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,13 +9,16 @@
...
+        <activity android:name=".SplashScreen"
+            >
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".MainActivity">
+
```
Comment (iadeelzafar) [inline app/src/main/AndroidManifest.xml]: Remove this extra line

**Comment by AminullahTajMuhammad** [comment]
@galihsatriawan Please add the description of PR

**Comment by iadeelzafar** [comment]
@galihsatriawan Please don't mark the conversations as resolved by yourself. That is supposed to be done by the reviewer.

**Comment by galihsatriawan** [comment]
Oh I'm so sorry

**Comment by galihsatriawan** [comment]
I have added the GIF

**Comment by galihsatriawan** [comment]
But I have a problem, I commit twice (for this, and MVVM Issue) but it merged with this pull request

**Comment by iadeelzafar** [comment]
@galihsatriawan No worries. We will review your solution for both tickets using this one Pull Request.

**Comment by iadeelzafar** [comment]
You need to resolve merge conflicts.
To do that,
* First add ChatAir's repo as a remote if you haven't done that already.
`git remote add upstream https://github.com/ChatAir/ChatAir-android.git`

* After that, pull the latest master.
`git pull upstream master`

After this, resolve the merge conflicts.

**Comment by iadeelzafar** [comment]
I just see that you're working on the local master branch. But you should be creating a new branch off top of master to work on each issue.
For example:
`git checkout master`
`git branch featureName`

**Comment by iadeelzafar** [comment]
If you're still having trouble with **GIT** then you may want to read a few articles on GIT.

**Comment by galihsatriawan** [comment]
Oh thanks that have reminded me, I will be more careful

**Comment by galihsatriawan** [comment]
Okay I have resolve the conflict and then could you give me a suggestion?, because in my fork (master) has changed with the last change, so when I create new branch right now that is not change anything

**Comment by AminullahTajMuhammad** [comment]
@galihsatriawan If you working on two issues in one pull request. The solution is that you have to delete the code of MVVM issue in the next commit.
### Here is the list of changes you have to do
- You have to follow the given designs.
- Use the indeterminant progress bar for the splash screen given in the below gif.

**Comment by galihsatriawan** [comment]
Okay this one

**Comment by AminullahTajMuhammad** [comment]
@galihsatriawan I told you that follow the designs. Read [This](https://github.com/ChatAir/ChatAir-android/pull/12#issuecomment-539660734) again and follow this.

**Comment by galihsatriawan** [comment]
Could you give me some suggestion maybe about my design?

**Comment by AminullahTajMuhammad** [comment]
@galihsatriawan Please use the indeterminate progress bar (2nd one in the previous image). You have used the definite progress bar instead.

**Comment by iadeelzafar** [comment]
This PR seems to be consuming a lot of time on a relevantly simple task. I think looking at the urgency, we should do it by ourselves @AminullahTajMuhammad . So, closing this.

**Comment by AminullahTajMuhammad** [comment]
@iadeelzafar I am already working on this UI.

**Comment by galihsatriawan** [comment]
I'm so sorry about it, there're some problems in a few day and I just worked with my laptop again

**Comment by iadeelzafar** [comment]
@galihsatriawan No worries. Its just that some tasks need to be completed urgently. That's why we had to close your PR. Feel free to join development anytime in the future.

**Pull request closed without merging**